modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/brainparcellationmesh/v0.1.0.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
This schema is already added to this [PR](https://github.com/INCF/neuroshapes/pull/96/files). Let fix and merge that one before the current PR.